HACK.md

```diff
@@ -80,6 +80,8 @@ The following is a list of environment variables you can use when running e2e te
 | `TEST_IMAGE_REPO`                  | `spec.output.image`            | Image repository for end-to-end tests               |
 | `TEST_IMAGE_REPO_SECRET`           | `spec.output.credentials.name` | Container credentials secret name                   |
 | `TEST_IMAGE_REPO_DOCKERCONFIGJSON` | _none_                         | JSON payload equivalent to `~/.docker/config.json`  |
+| `TEST_PARALLEL`                    | _none_                         | Number of tests to run in parallel, default is `8`. |
+| `TEST_TIMEOUT`                     | _none_                         | Timeout of the test run, default is `10m`.          |
```

Comment:
@otaviof asked for this here: https://github.com/redhat-developer/build/pull/163#discussion_r419368139. I had hard-coded to ten minutes before and do not have a strong opinion here.

But thinking about it, two use cases that come to my mind:

1. If you run on a smaller locally running Kubernetes cluster, you might be reducing the parallelism from 8 to maybe 4 and then ten minutes might not be enough anymore.
2. If you include "optional" specs, for example those with a private repository, then the run time is different. This may play a role should we face more scenarios in the future where certain specs are skipped if prereqs are not fulfilled.

Reply:
Discussed with @qu1queee; the performance of Travis is unforeseeable. We should therefore take a higher default.